.drone.star

```diff
@@ -1202,7 +1202,7 @@ def ocisService(type, tika_enabled = False, enforce_password_public_link = False
         "LDAP_GROUP_SUBSTRING_FILTER_TYPE": "any",
         "LDAP_USER_SUBSTRING_FILTER_TYPE": "any",
         "PROXY_ENABLE_BASIC_AUTH": True,
-        "WEB_ASSET_PATH": "%s/dist" % dir["web"],
+        "WEB_ASSET_CORE_PATH": "%s/dist" % dir["web"],
```

Comment:
need to bump ocis commit in web as well. Current commit doesn't have `WEB_ASSET_CORE_PATH` changes.

Reply:
yeah, thank you 
https://github.com/owncloud/web/pull/10554